middleware/verifyCategoryId.js

```diff
@@ -0,0 +1,25 @@
+import { Category } from '../models';
+
+export default async (req, res, next) => {
+  const { params: { id } } = req;
+
+  if (Number.isNaN(Number(id))
+|| !Number.isInteger(Number(id))
+|| Number(id) < 0) {
+    return res.status(400).json({
+      success: false,
+      errors: ['Invalid category id. Category id must be a positive integer.']
```

Comment:
Let's just send an error here that it's invalid. The user might not know what an integer is

Reply:
Yeah... sure.